blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DSimpleNumberIO.java

```diff
@@ -0,0 +1,27 @@
+package com.rackspacecloud.blueflood.io.datastax;
+
+import com.rackspacecloud.blueflood.io.serializers.metrics.RawSerDes;
+import com.rackspacecloud.blueflood.types.Rollup;
+
+import java.nio.ByteBuffer;
+
+/**
+ * This class holds the utility methods to read/write simple numbers
+ * using Datastax driver.
+ */
+public class DSimpleNumberIO extends DAbstractMetricIO {
+
+    private RawSerDes serDes = new RawSerDes();
+
+    @Override
+    protected ByteBuffer toByteBuffer( Object rollup ) {
```

Comment:
It's not a rollup (I'll change the parameter).  The serializer can be called on any ```Object```.